cypress/pages/freestandingPage.js

```diff
@@ -0,0 +1,152 @@
+export class freestandingPage{
+
+
+    freestandingPageSelectors = {
+        
+        colorTitle:"div[class='gf-block-title']",
+  //      color: "div[data-filter-id='98948']",
+        colorSearch:".gf-search",
+        colorBeechwood:"button[title='Beechwood']",
+        colorDarkOak:"button[title='Dark Oak']",
+        colorGray:"button[title='Gray']",
+        colorGreen:"button[title='Green']",
+        colorNavyBlue:"button[title='Navy Blue']",
+        colorWhite:"button[title='White']",
+
+        productTitle:"div.h4.spf-product-card__title>a",
+     //  t:'#HeaderMenu-sizes-' + chosenSize + '-inch', div.h4.spf-product-card__title a
+        
+        
+        sizeTitle:"div[class='gf-block-title']",
+        size: "div[data-filter-id='98949']",
+        size18: "button[title='18 Inch']",
+        size24: "button[title='24 Inch']",
+        size30: "button[title='30 Inch']",
+        size36: "button[title='36 Inch']",
+        size40: "button[title='40 Inch']",
+        size48: "button[title='48 Inch']",
+        size60: "button[title='60 Inch']",
+        size72: "button[title='72 Inch']",
+
+        price:"div[data-filter-id='98943']",
+        minPriceInput:"#min-98943",
+        maxPriceInput:"#max-98943",
+
+        searchProducts:".gf-controls-search-form",
+        graySelena:".h4 spf-product-card__title",
+        graySelenaDoubleSink:"#MediaGallery-template--21128380580113__main",
+
+
+        yourCart:"h1[class='title title--primary']",
+        yourCartContinueShopping:".underlined-link",
+        yourCartQuantity:".cart-item__quantity",
+        yourCartCheckOut:"button[class='cart__checkout-button button']",
+        yourCartPay:".dynamic-checkout__content",
+        yourCartFollowOnShop:"#NewsletterForm--sections--21128380875025__footer"
+        
+
+    }
+    
+    pickColor(color){
+        cy.get(this.freestandingPageSelectors.color).click()
+
+
+        
+
+            switch (color) {
+              case "Beechwood":
+                cy.get(this.freestandingPageSelectors.colorBeechwood).click()
+                break
+              case "Dark Oak":
+                cy.get(this.freestandingPageSelectors.colorDarkOak).click()
+                break
+              case "Gray":
+                cy.get(this.freestandingPageSelectors.colorGray).click()
+                break
+              case "Green":
+                cy.get(this.freestandingPageSelectors.colorGreen).click()
+                break
+              case "Navy Blue":
+                cy.get(this.freestandingPageSelectors.colorNavyBlue).click()
+                break
+              case "White":
+                cy.get(this.freestandingPageSelectors.colorWhite).click()
+                break
+              default:
+                throw new Error("There is no such color");
+            }
+
+    }
+   
+    chooseColor(color) {
+        cy.get('button[data-fvalue='+`${color}` + '] span').click();
```

Comment:
did this work? I remember there was an issue with this  selector? You can try with this:  ```
cy.get(
        `button[data-fvalue='+`${color}` + '] span`
      ).click({ force: true });
```